concepts/05 UI Components/Tooltip/02 Getting Started with Tooltip/00 Getting Started with Tooltip.md

```diff
@@ -0,0 +1,18 @@
+#include tutorials-intro-installationnote
+
+Tooltip is a UI component that allows you to display information over a specific element in your application.
+
+This tutorial guides you through the following steps: 
+
+- Add Tooltip to a page and configure core component settings.
+- Customize the appearance of Tooltip visual elements.
+
+The following live example demonstrates the result of this tutorial:
+
+<div class="simulator-desktop-container" data-view="/Content/Applications/25_1/GettingStartedWith/Tooltip/index.html, /Content/Applications/25_1/GettingStartedWith/Tooltip/index.js, /Content/Applications/25_1/GettingStartedWith/Tooltip/index.css"></div>
+
+Each section in this tutorial covers a single configuration step. You can find the complete source code in the following GitHub repository:
```

Comment:
At one point Ray asked not to use "covers" too often. So I'm just changing that part. 

```suggestion
Each section in this tutorial describes a single configuration step. You can find the complete source code in the following GitHub repository:
```